### DISCUSSION TITLE
Need help with URLs keywords

### DISCUSSION THREAD

**Discussion by zhori-7** [Flow-Launcher/Flow.Launcher#1099] [Q&A] [answered]
How can I assign a keyword to open a URL?
I just want it to open it without searching 
like I want to assign "drive" to open my google drive link main page without doing any search however currently I'm forced to supply a parameter to the keyword. 

I used to do this in another launcher called "Executor" but " left it bcz it looks outdated and I just need this feature in order to move to this launcher.

**Comment by jjw24** [comment]
Go to settings => plugins => find the Url plugin => then change the action keyword.

**Reply by zhori-7** [reply]
That would require me to paste google drive website link everytime.
It's not what I'm aiming for.

**Reply by jjw24** [reply]
You can't use the bookmark plugin to open the Url after bookmarking it?

**Comment by deefrawley** [comment, marked as answer]
If you didn't want to use a browser bookmark as jjw24 sugegsted, you could use the Plugin Runner plugin and create a "drive" command that runs the full path to your preferred browser with the Google Drive URL as the argument
